src/tools/ceph-dencoder/ceph_dencoder.cc

```diff
@@ -92,6 +112,13 @@ vector<DencoderPlugin> load_plugins()
 
 int main(int argc, const char **argv)
 {
+  auto args = argv_to_vec(argc, argv);
+  env_to_vec(args);
+
+  auto cct = global_init(NULL, args, CEPH_ENTITY_TYPE_CLIENT,
+                         CODE_ENVIRONMENT_UTILITY, 0);
+  common_init_finish(cct.get());
```

Comment:
this is sad... i deliberately avoid using `global_init()` if possible.

Reply:
Without this it's hard to newbees to make it work.